src/app/blog/components/post-detail/post-detail.component.html

```diff
@@ -35,21 +36,21 @@ <h1 data-aos="fade-in-up" data-aos-offset="100" class="text-primary">BLOG</h1>
       <div class="p-col">
         <div class="p-field p-grid">
           <div class="p-inputgroup">
-            <textarea pInputTextarea formControlName="msg" style="height:160px" placeholder="Message"></textarea>
+            <textarea pInputTextarea formControlName="msg" style="height:160px" placeholder="Message *"></textarea>
           </div>
         </div>
         <div *ngIf="msg.invalid && (msg.dirty || msg.touched)" class="p-col-12">
           <small *ngIf="msg.errors.required" class="p-invalid">Le Commentaire est requir.</small>
           <small *ngIf="msg.errors.minlength" class="p-invalid">Le Commentair doit etre de plus de 10 charactair
           </small>
         </div>
-      </div>
+      </div>&nbsp;&nbsp;
```

Comment:
Please use format to avoid those **&nbsp;**

Reply:
i am using this because i dont want to use scss file